docs/VALUES.md

```diff
@@ -0,0 +1,25 @@
+# Our mission
+
+`napari` aims to be the multi-dimensional image viewer for Python. We accomplish this by:
```

Comment:
Personally, I've viewed napari as more than just for image processing. I like the idea that people from many disciplines (e.g., geo/GIS, fluid mechanics) could find it useful. Do you think it is worthwhile to add a note about that here?

Reply:
like calling it `multi-dimensional data viewer`? i am open to that

Reply:
Yeah. I was going to suggest "array", but thanks to our many layer types, we can do more. However, I might suggest that we use array anyway, because my feeling is that our layers only really make sense in the context of an image. You wouldn't use napari for, say, a 3D scatterplot, even though it's possible. Should we restrict things a little here to ensure that development stays "on-mission"?

Reply:
I still have the word `image` in there, but have not forgotten about this comment. I prefer `array` to `data` as it is a bit more specific, though I worry it is a little jargony / python specific. I like `image` because it is more understandable to more people and I don't think puts people off using the tool (i.e. if you have complex simulation data you probably understand that you can look at your array with an image viewer), but if you have an image and have never heard of python or arrays you might not realize you can look at your image with an array viewer